docs/api.html

```diff
@@ -2,11 +2,12 @@
 layout: 2ColLeft
 title: API Reference
 sidebar: api.html
+edit: false
 ---
 
 {% markdown %}
 
-{% include api/api_overview.html %}
+{% include api/overview.html %}
```

Comment:
Not quite sure whats happening here but something broke, I see https://github.com/pouchdb/pouchdb/blob/master/docs/_includes/api/api_overview.html in the repo, but overview.html locally